src/bundle/Resources/public/scss/_preview.scss

```diff
@@ -76,32 +89,31 @@
             height: calc(100vh - 45px);
             display: block;
             margin: 0 auto;
-            transition: all .2s $ez-admin-transition;
+            transition: all 0.2s $ez-admin-transition;
         }
 
         &--tablet {
             background: url('/bundles/ezplatformadminui/img/preview-tablet.png') top center no-repeat;
-            height: 1250px;
-            margin-top: 1rem;
+            height: calculateRem(1250px);
```

Comment:
I'm wondering, do we need such big value for `height`? I mean, it will cause the scroll.

Reply:
Hi @sunpietro, I'd say yes if we want to show the full `preview-tablet.png` img which is preview's objective. In any case given that you are pointing this out `preview-tablet.png` height is 1287px, hence maybe the correct height would be 1290px.